Assets/Scripts/ViconNexusUnityStream/CustomHandScript.cs

```diff
@@ -670,11 +670,26 @@ public void SetHandJointRadius(XRHandJointID joint, float radius)
 
                 for (int i = XRHandJointIDUtility.ToIndex(XRHandJointID.BeginMarker); i < XRHandJointIDUtility.ToIndex(XRHandJointID.EndMarker); ++i)
                 {
-                    xrHandJointRadiiList.Add(new XRHandJointRadius() { joint = XRHandJointIDUtility.FromIndex(i), radius = 0 });
+                    xrHandJointRadiiList.Add(new() { joint = XRHandJointIDUtility.FromIndex(i), radius = 0 });
                 }
             }
+            xrHandJointRadiiList[joint.ToIndex()] = new() { joint = joint, radius = radius };
```

Comment:
I would avoid the shortened `new` if the type is not being defined in the same line. Same with above line change with `new`.